src/main/webapp/i18n/en/error.json

```
@@ -17,21 +17,21 @@
         "resultNotRated": "Error! Result must be rated.",
         "submissionMissing": "Error! The submission is not connected with the result any more. Your changes were rejected.",
         "resultAlreadyExists": "A result already exists for this student in this exercise.",
-        "scoreAndSuccessfulNotMatching": "Error! Only result with score 100% can be successful.",
-        "feedbackDetailTextNull": "Error! In case feedback is present, feedback detail text are mandatory.",
+        "scoreAndSuccessfulNotMatching": "Error! Only a result with a score of 100% can be successful.",
+        "feedbackDetailTextNull": "Error! If feedback is present, all elements must contain text.",
```

```suggestion
        "feedbackDetailTextNull": "Error! If feedback is present, it must contain feedback detail text.",
```
something along those lines, as **all elements** is not valid here. There is only one element in context, as a feedback has a single detailed text field.

---

Thanks for pointing that out!